css/css-ui-3/text-overflow-017.html

```diff
@@ -0,0 +1,37 @@
+﻿<!DOCTYPE html>
+<html class="reftest-wait">
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: interacting with the ellipsis</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#text-overflow">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem interact dom">
+<meta name="assert" content="Ellipsing must not affect dispatching of pointer events.">
```

Comment:
We have interop on this test, all browsers pass it.

However, we do not have full interop on whether or not the event also gets disptached to the span if the elided text is in a span, or to inbetween elements if there's any: Firefox does dispatch the event to all these when clicking on the ellipsis, but other browsers do not, and send it directly to the block element.

Eitherway, I've filled a bug on the spec (l3 & l4) https://github.com/w3c/csswg-drafts/issues/1637.

We may end up clarifying in level 3, but since it's otherwise getting ready to get to PR, at this point I would prefer to limit edits to correcting places where the spec explicitly contradicts (interoperable) behavior that browsers want to keep (if any such thing is found), and leave clarifications of undefined behavior for next level.

Regardless of which way we go, I think this test is correct. We will need more tests when we decide to clarify the sentence (or if we decide that it is already clear and requires one of the behaviors), and I'll be happy to add more tests when that's settled, but this should not block this PR.